portal-api/src/v1/api-response-types/utilisation-report-response-body.ts

```diff
@@ -4,8 +4,8 @@ export type UtilisationReportResponseBody = {
   id: number;
   bankId: string;
   status: UtilisationReportReconciliationStatus;
-  uploadedByUserId: string;
+  uploadedByUserId?: string;
```

Comment:
What is this change for?

Reply:
`uploadedByUserId` and `dateUploaded` are optional in the entity and so they will also be optional in the response